app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -1,246 +1,182 @@
-<form class="ui form {{if this.isLoading 'loading'}}" autocomplete="off" {{action 'move' 'forward' on='submit' preventDefault=true}} novalidate>
+<form class="ui form {{if this.isLoading 'loading'}}" autocomplete="off"
+  {{action 'move' 'forward' on='submit' preventDefault=true}} novalidate>
   <div class="field">
     <label class="required" for="name">{{t 'Event Name'}}</label>
-    <Input
-      @type="text"
-      @id="name"
-      @value={{this.data.event.name}} />
+    <Input @type="text" @id="name" @value={{this.data.event.name}} />
   </div>
   <div class="ui secondary pointing menu">
     {{#each this.locationMenuItems as |item|}}
-      <a role="button" href="#" class="{{if (eq item this.selectedLocationType) 'active'}} item" onclick={{action (mut this.selectedLocationType) item}}>
-        {{item}}
-      </a>
+    <a role="button" href="#" class="{{if (eq item this.selectedLocationType) 'active'}} item"
+      onclick={{action (mut this.selectedLocationType) item}}>
```

Comment:
on line 5 you have made a multiline tag to a single line and here a single line to multiline?

Reply:
make only the change that is required.